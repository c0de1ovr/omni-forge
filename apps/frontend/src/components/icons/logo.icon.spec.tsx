import { render } from '@testing-library/react';

import { Logo } from './logo.icon';

describe('logo icon', () => {
  it('should render logo icon correctly', () => {
    const { container } = render(<Logo />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should accept and use height', () => {
    const { container } = render(<Logo width={10} />);
    expect(container.firstChild).toMatchSnapshot();
  });

  it('should accept and use width', () => {
    const { container } = render(<Logo height={10} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
