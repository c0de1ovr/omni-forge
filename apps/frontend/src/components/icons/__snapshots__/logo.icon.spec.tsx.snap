// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logo icon should accept and use height 1`] = `
<svg
  fill="none"
  height="36"
  viewBox="0 0 32 32"
  width="10"
>
  <path
    clip-rule="evenodd"
    d="M17.6482 10.1305L15.8785 7.02583L7.02979 22.5499H10.5278L17.6482 10.1305ZM19.8798 14.0457L18.11 17.1983L19.394 19.4511H16.8453L15.1056 22.5499H24.7272L19.8798 14.0457Z"
    fill="currentColor"
    fill-rule="evenodd"
  />
</svg>
`;

exports[`logo icon should accept and use width 1`] = `
<svg
  fill="none"
  height="10"
  viewBox="0 0 32 32"
  width="36"
>
  <path
    clip-rule="evenodd"
    d="M17.6482 10.1305L15.8785 7.02583L7.02979 22.5499H10.5278L17.6482 10.1305ZM19.8798 14.0457L18.11 17.1983L19.394 19.4511H16.8453L15.1056 22.5499H24.7272L19.8798 14.0457Z"
    fill="currentColor"
    fill-rule="evenodd"
  />
</svg>
`;

exports[`logo icon should render logo icon correctly 1`] = `
<svg
  fill="none"
  height="36"
  viewBox="0 0 32 32"
  width="36"
>
  <path
    clip-rule="evenodd"
    d="M17.6482 10.1305L15.8785 7.02583L7.02979 22.5499H10.5278L17.6482 10.1305ZM19.8798 14.0457L18.11 17.1983L19.394 19.4511H16.8453L15.1056 22.5499H24.7272L19.8798 14.0457Z"
    fill="currentColor"
    fill-rule="evenodd"
  />
</svg>
`;
